scripts/export_samples.py

```diff
@@ -0,0 +1,18 @@
+"""Run all sample-genrating modules."""
+import os
+import importlib
+
+# gather all of the modules the contain sample files
+root_dir = os.path.dirname(os.path.abspath(__file__))
+module_names = os.listdir(root_dir)
+module_files = [os.path.join(root_dir, mod) for mod in module_names]
+sample_modules = []
+for mod, mod_file in zip(module_names, module_files):
+    if os.path.isfile(mod_file) and mod_file.endswith('.py') \
+            and mod.startswith('sample'):
+        sample_modules.append(mod[:-3])
+
+# execute each of the sample file modules
+for mod in sample_modules:
+    print('Generating samples for {}'.format(mod))
+    importlib.import_module(mod)
```

Comment:
Normally you don't import the modules like this and instead run it from the command line. You should use `if __name__ == '__main__' and call the functions that you want to be executed there. I will add a sample to `sample_model.py`.